src/api/java/baritone/api/Settings.java

```diff
@@ -1211,6 +1211,11 @@
      */
     public final Setting<Boolean> notificationOnMineFail = new Setting<>(true);
 
+    /**
+     * Creates a waypoint where you die
+     */
+    public final Setting<Boolean> waypointOnDeath = new Setting<>(true);
```

Comment:
If the goal is to prevent unnecessary waypoints `automaticWaypoints` for all automatically created waypoints would also be an option, but the only other type Baritone creates itself are beds and you usually don't come across beds on pvp servers (unless you are playing bedwars but then you shouldn't be using Baritone).

Reply:
I just want to disable death waypoints, I find all the other automatic waypoints verry usefull(and I cant think of a situation where you would want to disable other automatic waypoints tbh)

Reply:
Then leave it as is, I just wanted to mention it.

Reply:
Ye ik, but just so u know my goal is to stop death waypoints and not all waypoints :)